tools/loadtesting/terraform/redis.tf

```
@@ -11,8 +11,8 @@ resource "aws_elasticache_replication_group" "default" {
   port                          = "6379"
   snapshot_retention_limit      = 0
   automatic_failover_enabled    = true
-  at_rest_encryption_enabled    = false
-  transit_encryption_enabled    = false
+  at_rest_encryption_enabled    = true
```

encryption at rest is only available during initial creation of the redis cluster. Unfortunately this means for anyone applying terraform after it has been created, this will fail. What we can do is expose this as a variable, default it to false (since that is the existing default) and then set that variable to true and recreate the cluster in our environment.

---

```suggestion
  at_rest_encryption_enabled    = $some_tf_var ? false : true 
```

With the variable given a friendly description and through in `variables.tf`

---

Defaulting to true will recreate the Redis replication group. This results in about 10-15 minutes of downtime when I applied these changes locally.

---

@edwardsb then if it's 10-15m of downtime we should make it the default and communicate that. @gavinelder's suggestion is also nice, but I feel like since there is no benefit to having unencrypted redis beyond "you already have it that way" we should not even provide a toggle for people to use it that way